llm4pddl/utils.py

```diff
@@ -356,3 +358,58 @@ def str_to_identifier(x: str) -> str:
         https://stackoverflow.com/questions/5297448
     """
     return hashlib.md5(x.encode('utf-8')).hexdigest()
+
+
+def get_visualization(input_path: str, output_dir: str) -> None:
+    """Creates visualization of planner accuracy across all environments."""
+    df = pd.read_csv(input_path)
+    column_labels = []
+    _ = [column_labels.append(col) for col in df.columns]
+    env_column_index = column_labels.index('env')
+    approach_column_index = column_labels.index('approach')
+    accuracy_column_index = column_labels.index('success')
+    experiment_name_index = column_labels.index('experiment_id')
+
+    llm_multi_accuracies = {}
```

Comment:
can you think about a way to restructure this code so that if we want to add another approach in the future, all we have to do is add the name of that approach in one place?